model/keyvalue.go

```diff
@@ -27,6 +27,7 @@ const (
 	BinaryType = ValueType_BINARY
 
 	SpanKindKey     = "span.kind"
+	SpanHashKey     = "span.hash"
```

Comment:
let's move it into a separate group and use value `@jaeger@hash` - see #https://github.com/jaegertracing/jaeger/issues/6522